### PR TITLE
added a game component and added background

### PR DESCRIPTION
**Pull request by markmiw**

### CODE REVIEW

File: client/src/components/gamepage/Gamepage.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+function Gamepage({ user, game }) {
+  const renderGame = () => {
+    if (game === 'roulette') {
+      return <Roulette />;
+    }
+    if (game === 'scratch-ticket') {
+      return <ScratchTicket />;
+    }
+    return <Slots />;
+  };
```
Comment (Dalphus) [inline client/src/components/gamepage/Gamepage.jsx]: this function does not need to be a class method, move it outside the declaration of `Gamepage()`

File: client/src/components/gamepage/Gamepage.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <div className="game-container">
+      <Sidebar user={user} />
+      {renderGame()}
+    </div>
+  );
+}
+
+Gamepage.propTypes = {
+  user: PropTypes.object.isRequired,
```
Comment (Dalphus) [inline client/src/components/gamepage/Gamepage.jsx:28]: this line is throwing a linter error.
Our linter is configured to disallow generic objects as a prop type.
if you want to ignore this rule (which is what I would do, its an annoying rule) add
```
"react/forbid-prop-types": "off"
```
to the "rules" object in `.eslintrc.json`

File: client/src/components/gamepage/Gamepage.jsx (hunk 1 above)
Comment (Dalphus) [reply client/src/components/gamepage/Gamepage.jsx]: you may also want to use a `switch/case` block here, as the syntax is a little cleaner for this use case

**Review by BungaloBuce — APPROVED** [review]
Very nice!

**Review by Dalphus — CHANGES_REQUESTED** [review]
remove `.DS_Store` files and add `.DS_Store` to `.gitignore`

**Review by corneliusrenken — APPROVED** [review]

**Review by Dalphus — APPROVED** [review]

**Pull request merged**